kyrios.py

```diff
@@ -43,8 +44,16 @@ def installPackage(self,packageName,package,context,platformConfig):
             return
         print("DEBUG: packageManagerHomebrew: installing package '{}'".format(packageName))
         managedPackageName = platformConfig['packageName']
-        command = "brew install '{}'".format(managedPackageName)
+        options = platformConfig['options']
+        command = "brew {} install '{}'".format(options,managedPackageName)
         stdplus.run(command)
+        postInstall = platformConfig.get('postInstall',False)
+        if postInstall:
+            stdplus.run(postInstall)
+        if platformConfig.get('exitKyrios',False):
```

Comment:
When would this happen?

Reply:
In the case that you've just installed Docker ;-)